clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusClusterCachingAgent.groovy

```diff
@@ -375,16 +385,6 @@ class TitusClusterCachingAgent implements CachingAgent, CustomScheduledAgent, On
     }
   }
 
-
-  private void cacheTargetGroups(ServerGroupData data, Map<String, CacheData> targetGroups) {
```

Comment:
IntelliJ told me this wasn't being used

Reply:
I would think it should be... defer to @tomaslin here.